.github/workflows/ci.yml

```
@@ -13,7 +18,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-          fetch-depth: 0
+          persist-credentials: false # This will not store the git credentails on disk, only in memory
```

The removal of fetch depth seems unintentional
```suggestion
          fetch-depth: 0
          persist-credentials: false # This will not store the git credentials on disk, only in memory
```